collectors/python.d.plugin/alarms/alarms.chart.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+# Description: alarms netdata python.d module
+# Author: andrewm4894
+# SPDX-License-Identifier: GPL-3.0-or-later
+
+from json import loads
+
+from bases.FrameworkServices.UrlService import UrlService
+
+priority = 25
+update_every = 1
+
+status_map = {'UNINITIALIZED': 0 , 'UNDEFINED': 0, 'CLEAR': 0, 'WARNING': 1, 'CRITICAL': 2}
```

Comment:
Same comment as i made in readme section:
- same value for different statuses ('UNINITIALIZED', 'UNDEFINED')
- not all statuses (https://learn.netdata.cloud/docs/agent/health/reference#alarm-statuses)

If there is some logic behind this decision, please explain

Reply:
Logic explained above - i think chart would get too noisy with all status when the most important are CLEAR, WARNING, CRITICAL but that's just my take and open to ideas. From reading the docs it looks like maybe UNDEFINED is also an important status you might want to see- so was thinking maybe -1 for that perhaps.

Reply:
My point that if this collector tracks alarms statuses, then it should do it because that is his gist - collect and store current status whatever it is. At least by default - could be configurable.

@vlvkobal @thiagoftsm what do you think guys?

Reply:
I think every status should be distinct.

Reply:
I agree with @vlvkobal and @ilyam8 , all states have different meaning and if we decide to map them, we should map them with different values.